src/traction_modem/MemorySpace.cxxtest

```diff
@@ -55,8 +52,8 @@ static const MemorySpace::errorcode_t ERROR_AGAIN =
 
 TEST_F(MemorySpaceTest, Create)
 {
-    CvSpace cs(&g_service, &mTxFlow_, &mRxFlow_);
-    FirmwareSpace fs(&g_service, &mTxFlow_, &mRxFlow_);
+    CvSpace cs(&g_service, &link_);
```

Comment:
we could have made the Link object be the Service. Then these constructors become simpler and the stateflows natively get the service pointer.